<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewExperimentWindow</class>
 <widget class="QWidget" name="NewExperimentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>453</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>651</width>
    <height>453</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>651</width>
    <height>453</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Particles in Box</string>
  </property>
  <widget class="QGroupBox" name="groupParticleSettings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Particle settings</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>201</width>
      <height>174</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formParticleSettings">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="labelNRight">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Count (right side)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelParticleR">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Radius</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelNLeft">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Count (left side)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelVInit">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Initial speed</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelVLoss">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Speed loss factor</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="particleR">
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="vInit">
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="vLoss">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="nLeft">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="nRight">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupGeometry">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>201</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Box geometry</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>200</width>
      <height>209</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formBoxGeometry">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelBoxWidth">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelBoxHeight">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelBarrierX">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Barrier X</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelBarrierWidth">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Barrier width</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelHoleY">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Hole Y</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelHoleHeight">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Hole height</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="boxWidth">
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="boxHeight">
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="barrierX">
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="barrierWidth">
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="holeY">
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="holeHeight">
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupCollisionSettings">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>211</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Speed change after collision</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>211</width>
      <height>106</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formCollisionSettings">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelDeltaVTop">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Top</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelDeltaVBottom">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Bottom</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelDeltaVSide">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Sides</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="deltaVTop">
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="deltaVBottom">
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="deltaVSide">
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="buttonRun">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>420</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupMiscSettings">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>211</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Misc settings</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>211</width>
      <height>106</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formMiscSettings">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelG">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>g</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelSimulationTime">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Min. to simulate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelFps">
       <property name="minimumSize">
        <size>
         <width>115</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>115</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Frames/second</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="g">
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>9.800000000000001</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="fps">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="simulationTime">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>180</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupOutputFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Output file</string>
   </property>
   <widget class="QLineEdit" name="outputFile">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>25</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="buttonOutputFile">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>25</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupInputFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>631</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Or open an existing file</string>
   </property>
   <widget class="ClickableLabel" name="labelInputFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>601</width>
      <height>101</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Drag and Drop file here
or click to browse</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="ClickableLabel" name="labelClearInputFile">
    <property name="geometry">
     <rect>
      <x>563</x>
      <y>20</y>
      <width>50</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>src/clickablelabel.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
